docs/guide/cli-table.md

```diff
@@ -120,3 +120,4 @@
 | `--no-color` | Removes colors from the console output |
 | `--clearScreen` | Clear terminal screen when re-running tests during watch mode (default: `true`) |
 | `--standalone` | Start Vitest without running tests. File filters will be ignored, tests will be running only on change (default: `false`) |
+| `--blob <path>` | Paths to blob reports. If this options is used, Vitest won't run any tests, it will only report previously recorded tests |
```

Comment:
> Run vitest --blob=blob1.js --blob=blob2.js to report all test runs

Personally I don't really like this option name. I would prefer it to be something like `vitest --merge-blobs`, `vitest --merge-reports` or even as a command `vitest merge-reports ./reports/*`. 

@hi-ogawa what do you think?

Reply:
Yeah, passing `--blob ... --blob ...` looks a bit odd.
It looks like playwright's `merge-reports` is more opinionated and just accepts single directory  https://playwright.dev/docs/test-sharding#merge-reports-cli